dashboard-metrics-report-generator/src/main/resources/application.conf

```
@@ -19,8 +19,8 @@ collections {
   purposes = ${PURPOSES_COLLECTION_NAME}
   eservices = ${ESERVICES_COLLECTION_NAME}
 }
-party-management {
-  party-management-url = ${PARTY_MANAGEMENT_URL}
-  party-management-api-key = ${PARTY_MANAGEMENT_API_KEY}
+selfcare-v2-client {
+  url = ${SELFCARE_V2_CLIENT_URL}
+  api-key = ${SELFCARE_V2_CLIENT_API_KEY}
```

These are names used in other services
```suggestion
  url = ${SELFCARE_V2_URL}
  api-key = ${SELFCARE_V2_API_KEY}
```